api/octokit.ts

```diff
@@ -213,10 +213,14 @@ export class OctoKit implements GitHub {
 			})
 		).data;
 		const currentDate = new Date();
-		const possibleMilestones = allMilestones.filter(
-			(milestone) =>
-				new Date(milestone.due_on) > currentDate && currentDate > new Date(milestone.created_at),
-		);
+		const possibleMilestones = allMilestones
+			.filter(
+				(milestone) =>
+					new Date(milestone.due_on) > currentDate &&
+					currentDate > new Date(milestone.created_at) &&
+					!milestone.title.includes('Recovery'),
```

Comment:
Do we ever have lowercase Milestone? Not sure if we have to be careful of casing.

Reply:
I don't think so, quick skim suggests it has never happened before